kernel/mutex.c

```diff
@@ -114,7 +114,11 @@ int z_impl_k_mutex_lock(struct k_mutex *mutex, k_timeout_t timeout)
 
 	key = k_spin_lock(&lock);
 
-	if (likely((mutex->lock_count == 0U) || (mutex->owner == _current))) {
+	if (likely((mutex->lock_count == 0U) || (mutex->owner == _current) ||
```

Comment:
This file change seems unrelated to this commit and PR?

Reply:
Removed